pymc3/math.py

```diff
@@ -12,9 +12,92 @@
 from theano.tensor.nnet import sigmoid
 from theano.gof import Op, Apply
 import numpy as np
+import scipy.linalg
+from pymc3.theanof import floatX
+
 # pylint: enable=unused-import
 
 
+class Cholesky(Op):
+    """
+    Return a triangular matrix square root of positive semi-definite `x`.
+
+    This is a copy of the cholesky op in theano, that doesn't throw an
+    error if the matrix is not positive definite, but instead returns
+    nan.
```

Comment:
In fact, this is mostly a copy of an existing Theano op in tensor/slinalg.py I think the difference can be moved upstream.